app/assets/stylesheets/open_project_global/_variables.sass

```diff
@@ -182,3 +182,5 @@ $user-avatar-border-radius:                     50px !default
 $user-avatar-width:                             36px !default
 $user-avatar-mini-border-radius:                50px !default
 $user-avatar-mini-width:                        20px !default
+
+$select-element-padding:                        3px, 24px, 3px, 3px
```

Comment:
@NobodysNightmare COMMAS! – I was thinking one thing and typing one other _brain disconnect_